src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -1267,7 +1267,11 @@ private static bool HadLambdaConversionError(BindingDiagnosticBag diagnostics, A
                             isParams: false,
                             refKind: refArg);
 
-                        SymbolDistinguisher distinguisher = new SymbolDistinguisher(binder.Compilation, displayArg, UnwrapIfParamsArray(parameter, isLastParameter));
+                        SymbolDistinguisher distinguisher = new SymbolDistinguisher(
+                            binder.Compilation,
+                            displayArg,
+                            UnwrapIfParamsArray(parameter, isLastParameter),
+                            SymbolDisplayFormat.CSharpErrorMessageNoParameterNamesFormat);
```

Comment:
>SymbolDisplayFormat.CSharpErrorMessageNoParameterNamesFormat [](http://example.com/codeflow?start=28&length=60)

Did we confirm that including names for all other call sites is appropriate? Otherwise, should we flip the default around? I.e. use ```SymbolDisplayFormat.CSharpErrorMessageNoParameterNamesFormat``` by default and preserve the existing behavior?

Reply:
Given the comment on the ```SymbolDistinguisher```, it feels like it is primarily about types, and it always looks at parameter types in ```UnwrapSymbol```.

Reply:
That makes sense, thanks. Changed: 076240928ce01f06b5361f738df8feac566ab6cb